src/app/components/Parallels/ParallelsFields.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { extractParallels } from '../../utils/bibDetailsUtils';
+
+const ParallelsFields = ({ field, content = '', fieldIndex = 0, bib }) => {
+  if (!content) return null;
+
+  const parallel = extractParallels(bib, field);
+
+  return (
+    <>
+      {(parallel &&
+        parallel[fieldIndex] &&
+        parallel[fieldIndex]
+          .map((value, idx) => {
+            if (!value) return null;
+
+            return (
+              <span
+                key={`${idx}_${fieldIndex}_${value}`}
+                dir={unicodeDirection(value)}
+                style={{ display: 'block' }}
```

Comment:
It's the `a` tag. Seems to be the tag is set with `display: inline-flex`. This is causing the blocks (the text) to be inline. It appears we cannot use the `ParallelsField` in this manner. Instead the values to `LinkableField` should be parsed out and only a single value sent to `LinkableField` and ultimately `ParallelsField`. This way there wont be multiple values `(spans)` as children to the `a` tag.

This will require refactoring.